mmpose/datasets/transforms/formatting.py

```diff
@@ -200,14 +165,8 @@ def transform(self, results: dict) -> dict:
 
         # pack instance labels
         gt_instance_labels = InstanceData()
-        for key, packed_key in self.label_mapping_table.items():
+        for key, packed_key in results['label_mapping_table'].items():
```

Comment:
There may not be a 'label_mapping_table' key in the results if 'GenerateTarget' is not present in the pipeline